_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -248,6 +252,22 @@ This `autocomplete` attribute is inapplicable because it has the `off` value.
 <label>Friend's first name<input type="text" autocomplete="off"/></label>
 ```
 
+#### Inapplicable Example 8
+
+This `input` element has a [fixed value](#73f2c2:fixed-value) due to its `type` [attribute value][] of `submit`. `autocomplete` does not apply to Submit buttons.
+
+```html
+<input type="submit" autocomplete="off"/>
```

Comment:
This should not have an `autocomplete="off"` since it is another case for not being Applicable. We should test each condition separately.
Also, we should try and have an **invalid** `autocomplete` here, so that tools that incorrectly consider this as Applicable would Fail it rather than Pass it. In the implementation mappings, Inapplicable and Passed are almost the same, but a Fail/Inapplicable discrepancy raises a red flag (see last paragraph of [Test case design / Consistency with accessibility requirements](https://act-rules.github.io/pages/design/test-cases/#consistency-with-accessibility-requirements)).

Reply:
@Jym77,

> This should not have an autocomplete="off" since it is another case for not being Applicable. We should test each condition separately.

Replaced off with a valid value.

----

About 

> Also, we should try and have an invalid autocomplete here, so that tools that incorrectly consider this as Applicable would Fail it rather than Pass it. In the implementation mappings, Inapplicable and Passed are almost the same, but a Fail/Inapplicable discrepancy raises a red flag (see last paragraph of [Test case design / Consistency with accessibility requirements](https://act-rules.github.io/pages/design/test-cases/#consistency-with-accessibility-requirements)).

per HTML specs

> The [missing value default](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#missing-value-default) and the [invalid value default](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#invalid-value-default) are the [Text](https://html.spec.whatwg.org/multipage/input.html#text-(type=text)-state-and-search-state-(type=search)) state.

consequently, the 1st Failed Example is covering this scenario (invalid autocomplete)

> Failed Example 1
> [Open in a new tab](https://w3.org/WAI/content-assets/wcag-act-rules/testcases/73f2c2/2ed049a75aaa549c0ba477c5048f7f2bb34cb160.html)
> 
> This autocomplete [attribute value](https://www.w3.org/WAI/standards-guidelines/act/rules/73f2c2/#attribute-value) has an unknown term that is not a [correct autocomplete field](https://www.w3.org/WAI/standards-guidelines/act/rules/73f2c2/#correct-autocomplete-field).
> 
> `<label>Username<input autocomplete="badname"/></label>`

If instead the intent is to include elements that don't support the autocomplete attribute and move these scenarios in failing examples it means to change applicability and expectations.